transfers/transfer.py

```diff
@@ -419,13 +458,13 @@ def main(user, api_key, ts_uuid, ts_path, depth, am_url, ss_url, transfer_type,
         # TODO What inputs do we want?
         microservice = status_info.get('microservice', '')
         run_scripts('user-input',
-            microservice,  # Current microservice name
-            str(microservice != current_unit.microservice),  # String True or False if this is the first time at this wait point
-            status_info['path'],  # Absolute path
-            status_info['uuid'],  # SIP/Transfer UUID
-            status_info['name'],  # SIP/Transfer name
-            status_info['type'],  # SIP or transfer
-        )
+                    microservice,  # Current microservice name
+                    str(microservice != current_unit.microservice),  # String True or False if this is the first time at this wait point
+                    status_info['path'],  # Absolute path
+                    status_info['uuid'],  # SIP/Transfer UUID
+                    status_info['name'],  # SIP/Transfer name
+                    status_info['type'],  # SIP or transfer
+                    )
```

Comment:
More of a style/preference thing, but I think moving 'user-input' to the next line and leaving them all 4-space-indented would be better.